### DISCUSSION TITLE
Backport Armor Toughness

### DISCUSSION THREAD

**Discussion by MagmaSlime123** [Roadhog360/Et-Futurum-Requiem#79] [Ideas]
Meow.

**Comment by Roadhog360** [comment]
What does armor toughness even do? I never understood it.

**Reply by MagmaSlime123** [reply]
Minecraft Wiki might help.
https://fictionhorizon.com/armor-toughness-in-minecraft-explained/

**Reply by Roadhog360** [reply]
I still don't quite understand, it describes toughness as protection, but armor already has a protection value....

**Reply by MagmaSlime123** [reply]
Well that's how the wiki explains it, blame Mojang lol.

**Reply by Roadhog360** [reply]
That's not even an official Wiki, and Mojang doesn't make the official Wiki pages.

**Reply by BL00DKNIGHT** [reply]
as it seems armor toughness is kinda another health bar for the protection that armor gives, like that you need to do an certain damage to give an actual damage in the player hp.

i'm not good at making analogy but hope it helps
